plutus-ledger-constraints/src/Ledger/Constraints/OnChain/V1.hs

```diff
@@ -145,7 +150,7 @@ checkTxConstraintFun ScriptContext{scriptContextTxInfo} = \case
                 vh == vh' && valuePred val && datumPred d
             isMatch _ = False
         in
-        traceIfFalse "Le" -- "MustSpendScriptOutputWithMatchingDatumAndValue"
+        traceIfFalse "Lf" -- "MustSpendScriptOutputWithMatchingDatumAndValue"
```

Comment:
Hmm, why change instead of introducing a new code for new constraint? This way we break the backward compatibility and it may confuse.